src/backend/joanie/core/templates/issuers/contract_definition.css

```diff
@@ -0,0 +1,98 @@
+body {
+  font-family: "Hind", sans-serif;
```

Comment:
`Hind` is not a embedded system font.
I think for a start, you could use a system font instead to embedded a font in template. Take a look at which font is used currently into Howard templates for example.